interface/asset-manager.md

```diff
@@ -1,7 +1,13 @@
 # Asset Manager
 
 ## Overview 
-The **Asset Manager** is an organized collection of **Assets** that can be readily created, imported, deleted and updated. Below, you will find a depiction of the **Asset Manager** and its features.
+The **Asset Manager** is an organized collection of **Assets** that can be readily created, imported, deleted and updated. 
+
+In order to open the **Asset Manager**,  *right-click* on the *file menu bar* in Incari and *click* on **Asset Manager** from the drop-down list. This can be seen in the illustration below:
+
+![](../.gitbook/assets/interface/asset-manager/open-asset-manager.PNG) 
+
+Also, you will find a depiction of the **Asset Manager** and its features below:
```

Comment:
The word depiction, to me, implies that something is an interpretation rather than a representation. Maybe just remove this line altogether as the image is self-explanatory

Reply:
it's removed